bitnami/airflow/Chart.yaml

```diff
@@ -32,4 +32,4 @@ name: airflow
 sources:
   - https://github.com/bitnami/bitnami-docker-airflow
   - https://airflow.apache.org/
-version: 10.2.8
+version: 10.2.9
```

Comment:
In this case, since it is adding a new feature (new env. vars) we should increase the minor version, e.g. `10.2.8 -> 10.3.0`.

Reply:
Ah, good call.  I wasn't sure if this was an accidental oversite in the original and would be considered a bug fix.  Will do!